packages/devtools-core/README.md

```diff
@@ -0,0 +1,27 @@
+# Prodo Developer Tools
+
+This is the core package for the Prodo Development Tools.
+The tools are implemented in Prodo.
+
+## Installation
+
+Add the devtools-plugin to your model so that your app and the devtools can communicate properly.
+
+```
+import { createModel } from "@prodo/core";
+import devToolsPlugin from "@prodo/devtools-plugin";
+
+export interface State {
+}
+
+export const model = createModel<State>()
+  .with(devToolsPlugin);
+```
+
+## Usage
+
+Currently, you can use these devtools as a wrapper component around your Prodo application.
+
+### Wrapper Component
+
+All you have to do is install the plugin as described above.
```

Comment:
Is there any way to hide the devtools? If not there probably should be. Either by using a keybinding or passing in an option to `createStore` (like the logger).